river/ensemble/adaptive_random_forest.py

```diff
@@ -600,6 +660,21 @@ class AdaptiveRandomForestRegressor(BaseForest, base.Regressor):
     nominal_attributes
         [*Tree parameter*] List of Nominal attributes. If empty, then assume that
         all attributes are numerical.
+    attribute_observer
```

Comment:
Same comment as before.

Reply:
Updated in commit [f1eab02](https://github.com/creme-ml/creme/pull/379/commits/f1eab0274d21fedd3f599f8597e6a2f0230cbe26).